internal/packer/packer.go

```diff
@@ -1432,6 +1436,58 @@ func logic(cfg *config.Struct, programName string) error {
 			fmt.Printf("\n")
 		}
 
+	case cfg.Output.Type == config.OutputTypeGaf && cfg.Output.Path != "":
+		dir, err := os.MkdirTemp("", "gokrazy")
+		if err != nil {
+			return err
+		}
+		defer os.RemoveAll(dir)
+
+		tmpMBR, err = os.Create(path.Join(dir, "mbr.img"))
```

Comment:
Use filepath.Join instead of path.Join for consistency with the rest of the file

Reply:
Done